connection.js

```diff
@@ -692,12 +692,10 @@ InitOperation.prototype.onTimeout = function onTimeout(now) {
 };
 
 TChannelConnection.prototype.sendLazyErrorFrame =
-function sendLazyErrorFrame(reqFrame, codeString, message) {
+function sendLazyErrorFrame(id, tracing, codeString, message) {
```

Comment:
this refactor is an easy win, please pull it out first; it follows a prior refactor across v2/handler and connection